lc0/LC0VSProj/LC0VSProj.vcxproj

```diff
@@ -0,0 +1,156 @@
+    <ClInclude Include="..\src\selfplay\loop.h" />
+    <ClInclude Include="..\src\selfplay\tournament.h" />
+    <ClInclude Include="..\src\uciloop.h" />
+    <ClInclude Include="..\src\utils\bititer.h" />
+    <ClInclude Include="..\src\utils\cache-old.h" />
+    <ClInclude Include="..\src\utils\cache.h" />
+    <ClInclude Include="..\src\utils\commandline.h" />
+    <ClInclude Include="..\src\utils\cppattributes.h" />
+    <ClInclude Include="..\src\utils\exception.h" />
+    <ClInclude Include="..\src\utils\hashcat.h" />
+    <ClInclude Include="..\src\utils\mutex.h" />
+    <ClInclude Include="..\src\utils\optional.h" />
+    <ClInclude Include="..\src\utils\optionsdict.h" />
+    <ClInclude Include="..\src\utils\random.h" />
+    <ClInclude Include="..\src\utils\transpose.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <CudaCompile Include="..\src\neural\network_cudnn.cu" />
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{CBA46B8C-63B8-4F7F-A35D-E1464246DC72}</ProjectGuid>
+    <RootNamespace>LC0VSProj</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+  </PropertyGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
+    <ConfigurationType>Application</ConfigurationType>
+    <UseDebugLibraries>true</UseDebugLibraries>
+    <CharacterSet>MultiByte</CharacterSet>
+    <PlatformToolset>v141</PlatformToolset>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
+    <ConfigurationType>Application</ConfigurationType>
+    <UseDebugLibraries>false</UseDebugLibraries>
+    <WholeProgramOptimization>true</WholeProgramOptimization>
+    <CharacterSet>MultiByte</CharacterSet>
+    <PlatformToolset>v141</PlatformToolset>
+  </PropertyGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
+  <ImportGroup Label="ExtensionSettings">
+    <Import Project="$(VCTargetsPath)\BuildCustomizations\CUDA 9.1.props" />
+  </ImportGroup>
+  <ImportGroup Label="PropertySheets" Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
+  </ImportGroup>
+  <ImportGroup Label="PropertySheets" Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
+  </ImportGroup>
+  <PropertyGroup Label="UserMacros" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <LinkIncremental>true</LinkIncremental>
+    <IncludePath>$(ProjectDir)\..\src;$(VC_IncludePath);$(WindowsSDK_IncludePath);E:\Ankan\MachineLearning\CUDA_NN\7.1</IncludePath>
```

Comment:
remove hardcoding?

Reply:
done, but the developer will still need to make it point to location where cudnn library is present (no standard define for cudnn path :-/).